### DISCUSSION TITLE
Bazel container_pull of Python image: freeze major/minor version

### DISCUSSION THREAD

**Discussion by slapierre** [renovatebot/renovate#8381] [Request Help]
Hi,

I enabled Renovate in my Bazel monorepo where I use the `container_pull` rule to pull a Python 3.7.9 Docker image.

The bot opens 2 contradictory pull request: one to update the digest (awesome!) and another to update the image from Python 3.7.9 to 3.9.1 -- Update Python version? No thanks! :wink::

1. `chore(deps): update python_3.7.9_slim_buster:3.7.9-slim-buster docker digest to 64a8d50`
2. `chore(deps): update python_3.7.9_slim_buster docker tag to v3.9.1`

Is there a way to configure the behavior of `container_pull` to freeze the major and minor version, similar to what is documented in [Language Support :: Docker](https://docs.renovatebot.com/docker/)? 

At the moment we'll be on the lookout for such PRs and we'll close / won't merge them.

I created a small project (well... I recycled my repo from another discussion) where we can observe this behavior:

- https://github.com/slapierre/renovate-8297/pulls

Thanks!
Simon

**Comment by rarkins** [comment]
In Renovate terms it's not considered contradictory to have multiple PRs for the same dependency. e.g. just because a newer version exists, it doesn't mean you want to stop receiving digest updates for the current version. 

There are a few ways to achieve what you ask, here is one:


```json
{
  "packageRules": [{
    "managers": ["bazel"],
    "packagePatterns": ["^python"],
    "updateTypes": ["minor", "major"],
    "enabled": false
  }]
}
```

i.e. any bazel dependencies starting with "python" should disable minor and major updates. If you want to still get *patch* updates (e.g. to 3.7.10) despite your definition being called `python_3.7.9_slim_buster`, then also add this:

```json
{
  "packageRules": [{
    "managers": ["bazel"],
    "packagePatterns": ["^python"],
    "separateMinorPatch": true
  }]
}
```

**Comment by slapierre** [comment]
Thanks you @rarkins, much appreciate the swift reply!
